scene/resources/2d/tile_set.h

```diff
@@ -999,6 +999,7 @@ class TileData : public Object {
 	// Custom data.
 	void set_custom_data(String p_layer_name, Variant p_value);
 	Variant get_custom_data(String p_layer_name) const;
+	bool has_custom_data(String p_layer_name) const;
```

Comment:
```suggestion
	bool has_custom_data(const String &p_layer_name) const;
```
Let's make sure to pass by reference even if the others don't (but should)